mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +471,45 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _events_from_eeglab_raw(fname, event_id=dict()):
```

Comment:
I tried modeling it after the BV case as much as possible, to have some consistency across the readers. For example, it would have been trivial to directly return events, but this looks more modular to me.